spec/steady_state/attribute_spec.rb

```diff
@@ -120,6 +120,10 @@ def state
         expect(subject.state).to eq 'liquid'
       end
 
+      it 'includes a class method for all states' do
+        expect(steady_state_class.states.map(&:to_s)).to eq %w(solid liquid gas plasma)
```

Comment:
I believe that the way the `State` delegator wraps the values, they will respond to equality checks as if they are strings:

```suggestion
        expect(steady_state_class.states).to eq %w(solid liquid gas plasma)
```